examples/react/react-router/package.json

```diff
@@ -20,7 +21,11 @@
     "sort-by": "^1.2.0"
   },
   "devDependencies": {
+    "@types/react": "^18.2.45",
+    "@types/react-dom": "^18.2.18",
+    "@types/sort-by": "^1.2.3",
     "@vitejs/plugin-react": "^4.2.1",
+    "typescript": "5.2.2",
```

Comment:
@lachlancollins I think this would fail on main again because `sherif` doesn't run on PRs....

Reply:
Let me temporarily fix this by adding another task like format and knip!